java/operator/src/main/java/com/instaclustr/cassandra/operator/Operator.java

```diff
@@ -49,6 +44,9 @@
     static class OperatorOptions {
         @Option(names = {"-n", "--namespace"}, description = "")
         String namespace = "default";
+
+        @Option(names = {"-p", "--allow-pvc-cleanup"}, description = "")
+        Boolean allowPVCCleanups = false;
```

Comment:
Minor nit: Probably better to use the primitive `boolean` type rather than the boxed `Boolean` here (and where it's injected). `Boolean` is nullable.

Not a big issue here, but I always see usages of `Boolean` and other boxed types in fields or parameters as code smells. They're really only useful in collections or in the case when the value is "optional" (e.g., can be unset).

Reply:
For some reason I thought that primitives couldn't be injected...